scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -1224,6 +1224,31 @@ public Void doInTransaction(Session session) {
         });
     }
 
+    public void updateJobAndTasksStatuses(final InternalJob job, final Set<TaskId> taskIds, TaskStatus taskStatus) {
```

Comment:
Job and task can have only status. Isn't it better to call the method updateJobAndTasksStatus

Reply:
Yes, Thanks